rootfs-builder/rootfs.sh

```diff
@@ -225,7 +225,7 @@ if [ -n "${USE_DOCKER}" ] ; then
 		-v "${ROOTFS_DIR}":"/rootfs" \
 		-v "${script_dir}/../scripts":"/scripts" \
 		-v "${kernel_mod_dir}":"${kernel_mod_dir}" \
-		-v "${GOPATH}":"${GOPATH}" \
+		-v "${GOPATH%%:*}":"${GOPATH%%:*}" \
```

Comment:
Please fetch the first element of GOPATH and save it in a local variable. Then you can use the local variable here and in below pushd.

Reply:
ok. I will fetch the element and store it in GOPATH_LOCAL.